app/src/components/HomePage/HomePage.tsx

```diff
@@ -115,15 +123,13 @@ export default function HomePage({
   }, [cityYears]);
 
   const { data: inventoryOrgData, isLoading: isInventoryOrgDataLoading } =
-    useGetOrganizationForInventoryQuery(inventoryIdFromParam as string, {
+    useGetOrganizationForInventoryQuery(inventoryIdFromParam!, {
```

Comment:
why do we need the `!` here?

Reply:
Yes, this gives a type error otherwise. It will never be passed into the API as when it's null the skip condition will be active.